src/main/java/com/epam/ta/reportportal/binary/DataStoreService.java

```diff
@@ -45,6 +61,12 @@
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(DataStoreService.class);
 
+	private static final String ROOT_USER_PHOTO_DIR = "users";
```

Comment:
Let's try to decompose all logic following single responsibility principle.
we would have UserBinaryDataHandler, AttachmentHandler that uses a common DataStoreService